# A boolean value; true for open, false for gripping
bool open
